doc/doc.css

```diff
@@ -1,44 +1,226 @@
-.hero{
-    padding: 4rem;
+
+
+/* -------------   */
+
+@import url(../styles/main.css);
+
+
+/* ***** page layout *** */
+
+.page-layout{
     display: grid;
-    grid-template-columns: 6rem 1fr 1fr 2rem;
+    grid-template-columns: 1fr 3fr 0.3fr;
     grid-template-areas: 
-    ". left right .";
-    /* height: 80vh; */
-gap: 1rem;
+    "header  header header"
+    "sidebar  main   .";
+}
+
+
+/* ***** header ****** */
 
+.header{
+    grid-area: header;
+    padding: 1rem 2.8rem;
+    display: flex;
+    position: sticky;
+    top: 0;
+    z-index: 3;
+    align-items: center;
+    justify-content: space-between;
+    box-shadow: 0 10px 8px -8px #d6d6d6;
+    font-weight: 500;
+    background-color: var(--primary-color)
+}
 
+.header__logo{
+    display: flex;
+    margin-left: 7rem;
+}
+.header__logo > img{
+    width: 15rem;
+    height: 5.8rem
 }
 
-.hero > div {
-    padding: 1rem;
+.header__links{
+    list-style: none;
+    display: flex;
+    font-size: 2.9rem;
+    align-items: center;
+    margin-right: 6 rem;
+}
+
+.header__links li i{
+    color: black;
+}
 
+.header__links li i:hover{
+    cursor: pointer;
+    /* color: var(--primary-dark-color) */
+    color: var(--secondary-color)
 }
 
-.hero-left{
-    grid-area: left;
+
+/* ****** Sidebar ******* */
+
+.sidebar-main{
+    position: relative;
+    grid-area: sidebar;
+    
+}
+
+.sidebar{
+    padding: 3rem;
+    padding-left: 4.5rem;
+    padding-right: 3rem;
+    position: sticky;
+    overflow-y: auto;
+    top: 7.5rem;
+    height: calc(100vh - 8.125rem);
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    
+}
+
+.sidebar::-webkit-scrollbar {
+    display: none;
+  }
+
+.sidebar__links{
     display: flex;
     flex-direction: column;
+    align-items: center;
+    gap: 1rem;
+    position: relative;
+}
+
+.sidebar__links::after{
+    content: "";
+    position: absolute;
+    border-right: 2.5px solid var(--primary-color);
+    top: -14rem;
+    right: -0.9rem;
+    height: 145%;
+}
+
+.sidebar h3{
+    display: grid;
     justify-content: center;
-    align-items: flex-start;
-    width: 40vw;
- 
+    
 }
 
-.hero-left > a {
-    width: 18rem;
-    height: 4.5rem;
+.sidebar a{
+    width: 100%;
+    font-weight: 500;
     font-size: 1.8rem;
-    text-align: center;
-    line-height: 24px;
+    color: var(--dark-color);
+    margin-top: 0.5rem;
+    margin-bottom: 0.5rem;
+    padding: 0.5rem;
+    display: grid;
+    justify-content: center;
+}
+
+.sidebar li{
+    list-style: none;
+}
+
+.sidebar a:hover{
+    color: var(--primary-light-color);
+}
+
+/* ****** scroll margin ***** */
+ #alert-section,
```

Comment:
if possible try targeting using `sidebar__link >a` this can reduce the number of lines